browser-extension/common/src/popup/components/Popup/index.tsx

```diff
@@ -29,12 +30,55 @@ const Popup: React.FC = () => {
     chrome.runtime.sendMessage({ action: EXTENSION_MESSAGES.TOGGLE_EXTENSION_STATUS }, setIsExtensionEnabled);
   }, []);
 
+  const productHuntLaunchBanner = useMemo(() => {
+    const currentDate = new Date();
+
+    // PDT timezone is 12.5 hours behind IST
+    const startDate = new Date("2023-08-16T12:30:00"); // year-month-day T hour:minute:second
```

Comment:
I am not sure about using IST here. Can we sync this with UTC? Also, I think we should use epoch time instead of date string
```suggestion
    const startDate = new Date("2023-08-16T07:00:00Z"); // year-month-day T hour:minute:second
```

Reply:
new Date("2023-08-16T19:00:00Z")